x/wasm/types/tx.go

```diff
@@ -220,7 +220,7 @@ func (msg MsgUpdateAdmin) ValidateBasic() error {
 		return sdkerrors.Wrap(err, "new admin")
 	}
 	if strings.EqualFold(msg.Sender, msg.NewAdmin) {
-		return sdkerrors.Wrap(ErrInvalidMsg, "new admin is the same as the old")
+		return sdkerrors.Wrap(ErrInvalidMsg, "new admin is the same as the old admin (you input)")
```

Comment:
the user input the new admin, is it neccessary to modify the error message?

Reply:
If contract is empty，then return “new admin is the same as the old”. it maybe cause user blinded. so i change errmsg for user make it understand errmsg.